<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StepWindow</class>
 <widget class="QMainWindow" name="StepWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>344</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>668</width>
    <height>344</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>668</width>
    <height>344</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame_green">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>131</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(156, 204, 152);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_white">
    <property name="geometry">
     <rect>
      <x>119</x>
      <y>-1</y>
      <width>551</width>
      <height>351</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>551</width>
      <height>351</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>551</width>
      <height>351</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(250, 245, 245);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_title">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>291</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Configurações Degrau</string>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>140</y>
       <width>231</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>16</number>
      </property>
      <property name="leftMargin">
       <number>19</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>23</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_amplitude">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Amplitude</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_amplitude">
        <property name="toolTip">
         <string/>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit{
border: 2px solid black;
border-radius: 20px;
background: white;
};</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="step_button">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>270</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border: 2px solid black;
border-radius:10px;
}
QPushButton#pushButton:pressed {
    background-color: gray;
	color:white;
}</string>
     </property>
     <property name="text">
      <string>Gerar Sinal</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
